ahmetDev7/projecthr
# Infscp01 73 feature suppliers delete original

# Issue
jira link
https://project-hr.atlassian.net/jira/software/projects/INFSCP01/boards/3?issueParent=10109&selectedIssue=INFSCP01-73

## Description
The delete function works properly with the right response

## Type of change


## Steps to test

1. Run the program and run the .rest file
2. You may need to create a new supplier if there isn't any in your database
3. in such a situation, just copy the create function and create controller from the suppliers create branch and add it temporarily to your files
4. Create a supplier and check in your db if its there, if its there, just put the id of that supplier in the .rest file and run it
